classes/MusicAlbum/music_album.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '/MusicAlbum/item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(publish_date, on_spotify: false)
+    super(publish_date)
+    @on_spotify = on_spotify
+  end
+
```

Comment:
can_be_achieved? method is a private method.
to make this private simply we can add `private` before the method.